helm/minio/templates/_helper_create_user.txt

```diff
@@ -93,7 +93,7 @@ connectToMinio $scheme
 {{- range .Values.users }}
 echo {{ tpl .accessKey $global }} > $MINIO_ACCESSKEY_SECRETKEY_TMP
 {{- if .existingSecret }}
-cat /config/secrets/{{ tpl .secretKey $global }} >> $MINIO_ACCESSKEY_SECRETKEY_TMP
+cat /config/secrets/{{ tpl .accessKey $global }} >> $MINIO_ACCESSKEY_SECRETKEY_TMP
```

Comment:
Why should an accessKey is the secretKey?

Reply:
The secret named from `existingSecret` is mounted as `/config/secrets/`, and each included secret is available as a file. The file name is the key from the secret, i.e. the accessKey.

This is how it was behaving before: https://github.com/minio/minio/commit/ab9544c0d34a825a31cdd9eb9e251ef7faa1f1a3#diff-f85addb2f442309143dcb4608f33e1db48c17fae8ea5c8659e083f5ffaeae353L83

Reply:
That looks very confusing, the secret must be changed to ensure that secretKey is used for this .

Reply:
Yes. The example in `values.yaml` is not correct either:

https://github.com/minio/minio/blob/2393a13f86ab9f28f1fa4b692ef1e0a8a7a92164/helm/minio/values.yaml#L321-L324

We can use `existingSecretKey` to make this example correct:

```diff
-cat /config/secrets/{{ tpl .secretKey $global }} >> $MINIO_ACCESSKEY_SECRETKEY_TMP
+cat /config/secrets/{{ tpl .existingSecretKey $global }} >> $MINIO_ACCESSKEY_SECRETKEY_TMP 
```

Reply:
@harshavardhana I've updated the PR with `.existingSecretKey`.